charts/oes/charts/spinnaker/templates/configmap/halyard-init-script.yaml

```diff
@@ -68,10 +68,12 @@ data:
     if [ -f /tmp/spinnaker/.hal/default/profiles/fiat-local.yml ]; then
     sed -i  s/RELEASE_NAME/{{ .Release.Name }}/g /tmp/spinnaker/.hal/default/profiles/fiat-local.yml
     fi
+    sed -i  s/SPINNAKER_NAMESPACE/${SPINNAKER_NAMESPACE}/ /tmp/spinnaker/.hal/default/profiles/orca-local.yml
```

Comment:
we don't have to even set this as an environment variable. this can can be {{ .Release.Namespace }}